### ISSUE TITLE
Some conflict with symfony/console while installing laravel/installer via Composer

### ISSUE THREAD

**Issue by Albert221** [laravel/installer#116]
```
$ composer --version
Composer version 1.10.6 2020-05-06 10:28:10
$ php -v
PHP 7.4.5 (cli) (built: Apr 14 2020 16:17:34) ( ZTS Visual C++ 2017 x64 )
Copyright (c) The PHP Group
Zend Engine v3.4.0, Copyright (c) Zend Technologies
```

```
$ composer global require laravel/installer
Changed current directory to C:/Users/walbe/AppData/Roaming/Composer
Using version ^3.0 for laravel/installer
./composer.json has been updated
Loading composer repositories with package information
Updating dependencies (including require-dev)
Your requirements could not be resolved to an installable set of packages.

  Problem 1
    - Conclusion: don't install laravel/installer v3.0.1
    - Conclusion: don't install laravel/installer v3.0.0|remove symfony/console v3.3.5
    - Conclusion: don't install laravel/installer v3.0.0|don't install symfony/console v3.3.5
    - Installation request for laravel/installer ^3.0 -> satisfiable by laravel/installer[v3.0.0, v3.0.1].
    - laravel/installer v3.0.0 requires symfony/console ^4.0|^5.0 -> satisfiable by symfony/console[v4.0.0, v4.0
.1, v4.0.10, v4.0.11, v4.0.12, v4.0.13, v4.0.14, v4.0.15, v4.0.2, v4.0.3, v4.0.4, v4.0.5, v4.0.6, v4.0.7, v4.0.8
, v4.0.9, v4.1.0, v4.1.1, v4.1.10, v4.1.11, v4.1.12, v4.1.2, v4.1.3, v4.1.4, v4.1.5, v4.1.6, v4.1.7, v4.1.8, v4.
1.9, v4.2.0, v4.2.1, v4.2.10, v4.2.11, v4.2.12, v4.2.2, v4.2.3, v4.2.4, v4.2.5, v4.2.6, v4.2.7, v4.2.8, v4.2.9,
v4.3.0, v4.3.1, v4.3.10, v4.3.11, v4.3.2, v4.3.3, v4.3.4, v4.3.5, v4.3.6, v4.3.7, v4.3.8, v4.3.9, v4.4.0, v4.4.1
, v4.4.2, v4.4.3, v4.4.4, v4.4.5, v4.4.6, v4.4.7, v4.4.8, v5.0.0, v5.0.1, v5.0.2, v5.0.3, v5.0.4, v5.0.5, v5.0.6
, v5.0.7, v5.0.8].
    - Can only install one of: symfony/console[v4.0.0, v3.3.5].
    - Can only install one of: symfony/console[v4.0.1, v3.3.5].
    - Can only install one of: symfony/console[v4.0.10, v3.3.5].
    - Can only install one of: symfony/console[v4.0.11, v3.3.5].
    - Can only install one of: symfony/console[v4.0.12, v3.3.5].
    - Can only install one of: symfony/console[v4.0.13, v3.3.5].
    - Can only install one of: symfony/console[v4.0.14, v3.3.5].
    - Can only install one of: symfony/console[v4.0.15, v3.3.5].
    - Can only install one of: symfony/console[v4.0.2, v3.3.5].
    - Can only install one of: symfony/console[v4.0.3, v3.3.5].
    - Can only install one of: symfony/console[v4.0.4, v3.3.5].
    - Can only install one of: symfony/console[v4.0.5, v3.3.5].
    - Can only install one of: symfony/console[v4.0.6, v3.3.5].
    - Can only install one of: symfony/console[v4.0.7, v3.3.5].
    - Can only install one of: symfony/console[v4.0.8, v3.3.5].
    - Can only install one of: symfony/console[v4.0.9, v3.3.5].
    - Can only install one of: symfony/console[v4.1.0, v3.3.5].
    - Can only install one of: symfony/console[v4.1.1, v3.3.5].
    - Can only install one of: symfony/console[v4.1.10, v3.3.5].
    - Can only install one of: symfony/console[v4.1.11, v3.3.5].
    - Can only install one of: symfony/console[v4.1.12, v3.3.5].
    - Can only install one of: symfony/console[v4.1.2, v3.3.5].
    - Can only install one of: symfony/console[v4.1.3, v3.3.5].
    - Can only install one of: symfony/console[v4.1.4, v3.3.5].
    - Can only install one of: symfony/console[v4.1.5, v3.3.5].
    - Can only install one of: symfony/console[v4.1.6, v3.3.5].
    - Can only install one of: symfony/console[v4.1.7, v3.3.5].
    - Can only install one of: symfony/console[v4.1.8, v3.3.5].
    - Can only install one of: symfony/console[v4.1.9, v3.3.5].
    - Can only install one of: symfony/console[v4.2.0, v3.3.5].
    - Can only install one of: symfony/console[v4.2.1, v3.3.5].
    - Can only install one of: symfony/console[v4.2.10, v3.3.5].
    - Can only install one of: symfony/console[v4.2.11, v3.3.5].
    - Can only install one of: symfony/console[v4.2.12, v3.3.5].
    - Can only install one of: symfony/console[v4.2.2, v3.3.5].
    - Can only install one of: symfony/console[v4.2.3, v3.3.5].
    - Can only install one of: symfony/console[v4.2.4, v3.3.5].
    - Can only install one of: symfony/console[v4.2.5, v3.3.5].
    - Can only install one of: symfony/console[v4.2.6, v3.3.5].
    - Can only install one of: symfony/console[v4.2.7, v3.3.5].
    - Can only install one of: symfony/console[v4.2.8, v3.3.5].
    - Can only install one of: symfony/console[v4.2.9, v3.3.5].
    - Can only install one of: symfony/console[v4.3.0, v3.3.5].
    - Can only install one of: symfony/console[v4.3.1, v3.3.5].
    - Can only install one of: symfony/console[v4.3.10, v3.3.5].
    - Can only install one of: symfony/console[v4.3.11, v3.3.5].
    - Can only install one of: symfony/console[v4.3.2, v3.3.5].
    - Can only install one of: symfony/console[v4.3.3, v3.3.5].
    - Can only install one of: symfony/console[v4.3.4, v3.3.5].
    - Can only install one of: symfony/console[v4.3.5, v3.3.5].
    - Can only install one of: symfony/console[v4.3.6, v3.3.5].
    - Can only install one of: symfony/console[v4.3.7, v3.3.5].
    - Can only install one of: symfony/console[v4.3.8, v3.3.5].
    - Can only install one of: symfony/console[v4.3.9, v3.3.5].
    - Can only install one of: symfony/console[v4.4.0, v3.3.5].
    - Can only install one of: symfony/console[v4.4.1, v3.3.5].
    - Can only install one of: symfony/console[v4.4.2, v3.3.5].
    - Can only install one of: symfony/console[v4.4.3, v3.3.5].
    - Can only install one of: symfony/console[v4.4.4, v3.3.5].
    - Can only install one of: symfony/console[v4.4.5, v3.3.5].
    - Can only install one of: symfony/console[v4.4.6, v3.3.5].
    - Can only install one of: symfony/console[v4.4.7, v3.3.5].
    - Can only install one of: symfony/console[v4.4.8, v3.3.5].
    - Can only install one of: symfony/console[v5.0.0, v3.3.5].
    - Can only install one of: symfony/console[v5.0.1, v3.3.5].
    - Can only install one of: symfony/console[v5.0.2, v3.3.5].
    - Can only install one of: symfony/console[v5.0.3, v3.3.5].
    - Can only install one of: symfony/console[v5.0.4, v3.3.5].
    - Can only install one of: symfony/console[v5.0.5, v3.3.5].
    - Can only install one of: symfony/console[v5.0.6, v3.3.5].
    - Can only install one of: symfony/console[v5.0.7, v3.3.5].
    - Can only install one of: symfony/console[v5.0.8, v3.3.5].
    - Installation request for symfony/console (locked at v3.3.5) -> satisfiable by symfony/console[v3.3.5].


Installation failed, reverting ./composer.json to its original content.
```

**Comment by Albert221** [comment]
Nevermind, `mediawiki/minus-x` I had installed globally too had a dependence on `symfony/console` v3...

**Comment by taylankasap** [comment]
For future visitors, use `composer global show` to see what else is installed globally so you can refresh your memory.

**Comment by franckDev21** [comment]
- Root composer.json requires brackets/craftable-installer ^2.1 -> satisfiable by brackets/craftable-installer[v2.1.0].
    - brackets/craftable-installer v2.1.0 requires symfony/console ~3.0|~4.0|~5.0 -> found symfony/console[v3.0.0, ..., v3.4.47, v4.0.0, ..., v4.4.49, v5.0.0, ..., v5.4.24] but the package is fixed to v6.3.0 (lock file version) by a partial update and that version does not match. Make sure you list it as an argument for the update command.

Use the option --with-all-dependencies (-W) to allow upgrades, downgrades and removals for packages currently locked to specific versions.
You can also try re-running composer require with an explicit version constraint, e.g. "composer require brackets/craftable-installer:*" to figure out if any version is installable, or "composer require brackets/craftable-installer:^2.1" if you know which you need.
